### PR TITLE
12.0 mig beesdoo stock

### PR DESCRIPTION
**Pull request by NadTr**

### CODE REVIEW

File: beesdoo_stock/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "author": "Coop IT Easy SCRLfs",
+    "license": "AGPL-3",
+    "website": "www.coopiteasy.be",
+    "description": """
+        Enable action on multiple products of a stock receipt
+    """,
+    "data": [
+        'views/stock_view.xml',
+    ],
+    'installable': False,
```
Comment (vvrossem) [inline beesdoo_stock/__manifest__.py:19]: [Switch the installable key to True](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0#tasks-to-do-in-the-migration)

**Review by vvrossem — CHANGES_REQUESTED** [review]
Hi @NadTr ! Thank you for you work on the migration :confetti_ball: 
Some changes to make on the `__manifest__.py` and on `stock_view.xml`

File: beesdoo_stock/views/stock_view.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_picking_form_inherited" model="ir.ui.view">
+        <field name="name">stock.picking.form.inherit</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form"></field>
+        <field name="arch" type="xml">
+           <button name="do_new_transfer" position="after">
```
Comment (vvrossem) [inline beesdoo_stock/views/stock_view.xml:8]: `do_new_transfer` is now named `button_validate` in 12.0

**Comment by vvrossem** [comment]
Hi @NadTr, I've noticed another change to make. Cheers :clinking_glasses:

**Comment by remytms** [comment]
See #144

**Pull request closed without merging**